### PR TITLE
media-sound/qpwgraph: new package, add 0.5.2, 9999

### PR DESCRIPTION
**Pull request by AdelKS**
Hello,

Here you go @thesamesam  !

### CODE REVIEW

File: media-sound/qpwgraph/qpwgraph-0.5.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+inherit cmake xdg
+
+DESCRIPTION="PipeWire Graph Qt GUI Interface"
+HOMEPAGE="https://gitlab.freedesktop.org/rncbc/qpwgraph"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/rncbc/qpwgraph"
+else
+	SRC_URI="https://gitlab.freedesktop.org/rncbc/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.bz2 -> ${P}.tar.bz2"
```
Comment (thesamesam) [inline media-sound/qpwgraph/qpwgraph-0.5.2.ebuild:15]: I don't see much point in renaming it:
```suggestion
	SRC_URI="https://gitlab.freedesktop.org/rncbc/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.bz2"
```

It's easier to search for upstream archives etc without the rename.

File: media-sound/qpwgraph/qpwgraph-0.5.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		dev-qt/qtxml:5
+	)
+	qt6? (
+		dev-qt/qtbase:6[gui,widgets,xml,network]
+		dev-qt/qtsvg:6
+	)
+"
+RDEPEND="${DEPEND}"
+
+[[ ${PV} != "9999" ]] && S="${WORKDIR}/${PN}-v${PV}"
```
Comment (thesamesam) [inline media-sound/qpwgraph/qpwgraph-0.5.2.ebuild:46]: I'd prefer you put this directly under SRC_URI instead

**Review by thesamesam — COMMENTED** [review]
minor tweaks then good to go

File: media-sound/qpwgraph/qpwgraph-0.5.2.ebuild (hunk 1 above)
Comment (AdelKS) [reply media-sound/qpwgraph/qpwgraph-0.5.2.ebuild]: done!

File: media-sound/qpwgraph/qpwgraph-0.5.2.ebuild (hunk 2 above)
Comment (AdelKS) [reply media-sound/qpwgraph/qpwgraph-0.5.2.ebuild]: done!

**Pull request closed without merging**